### PR TITLE
Override default Ingress tls secretName

### PR DESCRIPTION
**Pull request by Sharathmk99**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
In our case we have Internal TLS certificate and created under different secret name under Liqo namespace. This PR changes will allow us to override default secret name generation from ChartName-auth with user provided secret name.

Fixes #(issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [x] Locally

### CODE REVIEW

File: deployments/liqo/README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 | auth.ingress.enable | bool | `false` | Enable/disable the creation of the Ingress resource. |
 | auth.ingress.host | string | `""` | Set the hostname for your ingress. |
 | auth.ingress.port | int | `443` | Set port for your ingress. |
+| auth.ingress.tlsSecretName | string | `""` | Override default(ChartName-auth) tls secretName |
```
Comment (frisso) [inline deployments/liqo/README.md:16]:
```suggestion
| auth.ingress.tlsSecretName | string | `""` | Override default (ChartName-auth) tls secretName |
```

File: deployments/liqo/values.yaml [hunk 2]
```diff
@@ -343,6 +343,8 @@ auth:
     class: ""
     # -- Set port for your ingress.
     port: 443
+    # -- Override default(ChartName-auth) tls secretName
```
Comment (frisso) [inline deployments/liqo/values.yaml:346]:
```suggestion
    # -- Override default (ChartName-auth) tls secretName
```

**Review by frisso — APPROVED** [review]

**Comment by Sharathmk99** [comment]
Made necessary changes. Please review

**Pull request merged**